services/business/istio_validations.go

```diff
@@ -42,6 +44,41 @@ func (in *IstioValidationsService) GetServiceValidations(namespace, service stri
 	return runObjectCheckers(objectCheckers), nil
 }
 
+// GetDeploymentsValidations returns an IstioValidations object with all the checks found when running
+// all the enabled checkers.
+func (in *IstioValidationsService) GetDeploymentsValidations(namespace string) (models.IstioValidations, error) {
+	// Get all the Istio objects from a Namespace and deployment
+	istioDetails, err := in.k8s.GetDeployments(namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	objectCheckers := []ObjectChecker{
+		checkers.DeploymentChecker{Deployments: istioDetails},
+	}
+
+	// Get groupal validations for same kind istio objects
+	return runObjectCheckers(objectCheckers), nil
+}
+
+// GetDeploymentValidations returns an IstioValidations object with all the checks found when running
+// all the enabled checkers.
+func (in *IstioValidationsService) GetDeploymentValidations(namespace, deployment string) (models.IstioValidations, error) {
+	// Get all the Istio objects from a Namespace and deployment
+	istioDetails, err := in.k8s.GetDeployment(namespace, deployment)
```

Comment:
hey, really small thing, nitpicking. I would change this var's name, because it doesn't hold istio information, it is just kubernetes information. I would go to something closer to deployment concepts.  <3